### ISSUE TITLE
Possible error in docs

### ISSUE THREAD

**Issue by bmblb** [bryntum/chronograph#10]
Shouldn't it be `graph4.write(identifier15, 50)` ?
https://github.com/bryntum/chronograph/blob/0db8d13b9b9cb733f5c5fe17c698f6aa23eb5994/src/chrono/Effect.ts#L69

**Comment by SamuraiJack** [comment]
Nope, `identifier15` is changed because the `max` identifier has changed (`max` is an upper bound for `identifier15`)
